diff --git a/esm/getDefaultExtensions.js b/esm/getDefaultExtensions.js
index f1e89ddf88025c6267cae898b850b4efd7b5a58c..f21cd94bfab863ab71aedbb3568541d0554ad212 100644
--- a/esm/getDefaultExtensions.js
+++ b/esm/getDefaultExtensions.js
@@ -3,9 +3,9 @@ import { basicSetup } from '@uiw/codemirror-extensions-basic-setup';
 import { EditorView, keymap, placeholder } from '@codemirror/view';
 import { oneDark } from '@codemirror/theme-one-dark';
 import { EditorState } from '@codemirror/state';
-import { defaultLightThemeOption } from './theme/light';
+import { defaultLightThemeOption } from './theme/light.js';
 export * from '@codemirror/theme-one-dark';
-export * from './theme/light';
+export * from './theme/light.js';
 export var getDefaultExtensions = function getDefaultExtensions(optios) {
   if (optios === void 0) {
     optios = {};
diff --git a/esm/index.js b/esm/index.js
index 2a719d319ef04e0c8a7400e914e32c1fd7be2e02..0175253d7b6e467d71515459c9c07abce77166b5 100644
--- a/esm/index.js
+++ b/esm/index.js
@@ -2,14 +2,14 @@ import _extends from "@babel/runtime/helpers/extends";
 import _objectWithoutPropertiesLoose from "@babel/runtime/helpers/objectWithoutPropertiesLoose";
 var _excluded = ["className", "value", "selection", "extensions", "onChange", "onStatistics", "onCreateEditor", "onUpdate", "autoFocus", "theme", "height", "minHeight", "maxHeight", "width", "minWidth", "maxWidth", "basicSetup", "placeholder", "indentWithTab", "editable", "readOnly", "root", "initialState"];
 import React, { useRef, forwardRef, useImperativeHandle } from 'react';
-import { useCodeMirror } from './useCodeMirror';
+import { useCodeMirror } from './useCodeMirror.js';
 import { jsx as _jsx } from "react/jsx-runtime";
 export * from '@codemirror/view';
 export * from '@codemirror/state';
 export * from '@uiw/codemirror-extensions-basic-setup';
-export * from './useCodeMirror';
-export * from './getDefaultExtensions';
-export * from './utils';
+export * from './useCodeMirror.js';
+export * from './getDefaultExtensions.js';
+export * from './utils.js';
 var ReactCodeMirror = /*#__PURE__*/forwardRef((props, ref) => {
   var {
       className,
diff --git a/esm/useCodeMirror.js b/esm/useCodeMirror.js
index eab503a65f7cd2b23584e77ab0da09867d7e7d88..ee8a5dcf632f50d222a2b1c152b1bde585714954 100644
--- a/esm/useCodeMirror.js
+++ b/esm/useCodeMirror.js
@@ -1,8 +1,8 @@
 import { useEffect, useState } from 'react';
 import { Annotation, EditorState, StateEffect } from '@codemirror/state';
 import { EditorView } from '@codemirror/view';
-import { getDefaultExtensions } from './getDefaultExtensions';
-import { getStatistics } from './utils';
+import { getDefaultExtensions } from './getDefaultExtensions.js';
+import { getStatistics } from './utils.js';
 var External = Annotation.define();
 var emptyExtensions = [];
 export function useCodeMirror(props) {
